File: 01-super-basic-questions/11-sort-array/solution.go
package main

import (
	"fmt"
	"sort"
)

func main() {

	var arrSize int
	fmt.Println("Enter the size of the array: ")
	fmt.Scan(&arrSize)

	arr := make([]int, arrSize)

	fmt.Printf("Enter %d elements for the array: ", arrSize)
	for i := 0; i < arrSize; i++ {
		fmt.Scan(&arr[i])
	}

	sort.Ints(arr)

	fmt.Println("Sorted array: ", arr)

}

File: 01-super-basic-questions/11-sort-array/solution.java
import java.util.Scanner;
import java.util.Arrays;

class SortArray{
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int arrSize;
        arrSize = sc.nextInt();

        int[] arr = new int[arrSize];

        for (int i = 0; i < arrSize; i++){
            arr[i] = sc.nextInt();
        }

        Arrays.sort(arr);

        System.out.println("Sorted array: ");
        for (int num: arr){
            System.out.print(num + " ");
        }

    }
}